guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md

```diff
@@ -0,0 +1,65 @@
+---
+group: release-notes
+title: Magento Open Source 2.1.18 Release Notes
+---
+
+
+
+*Release notes published on June 25, 2019.*
+
+
+{:.bs-callout .bs-callout-note}
+The Magento Open Source 2.1.18 software release marks the final supported software release for Magento Open Source version 2.1.x. Magento Open Source 2.1.x will no longer receive security updates or product quality fixes now that its support window has expired.
+
+To maintain the performance and security of your site, we advise you to upgrade to the latest version of Magento Open Source. We encourage you to reach out to your partners, developers, or Magento Customer Success contact for more information on upgrading your site.
+
+
+We are pleased to present {{site.data.var.ce}} 2.1.18. This release includes multiple enhancements to product security. 
+
+
+This release include security enhancements that help close cross-site scripting, arbitrary code execution, and sensitive data disclosure vulnerabilities as well as other security issues. No confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. We recommend that you upgrade your Magento software to the latest version as soon as possible.
```

Comment:
It seems like the "This release..." paragraph might fit better under the highlights section.  

Also, in the opening paragraph of the release notes, you say: ``` we advise you to upgrade to the latest version of Magento Open Source. ```

This info is repeated in the last sentence: We recommend that you upgrade ....

Reply:
Will move section